README.md

```diff
@@ -7,24 +7,6 @@ The MSAL library for .NET is part of the [Microsoft identity platform for develo
 | [Conceptual documentation](https://aka.ms/msalnet) | [Getting Started](https://learn.microsoft.com/entra/msal/dotnet/getting-started/choosing-msal-dotnet) | [Sample Code](https://aka.ms/aaddevsamplesv2) | [API Reference](https://learn.microsoft.com/dotnet/api/overview/) | [Support](README.md#community-help-and-support) | [Feedback](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues)
 | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
 
-## NuGet packages
-
-**Microsoft.Identity.Client**  
-[![NuGet](https://img.shields.io/nuget/v/Microsoft.Identity.Client.svg?style=flat-square&label=nuget&colorB=00b200)](https://www.nuget.org/packages/Microsoft.Identity.Client/)
```

Comment:
this is one way after every release I verify if the NuGets have been updated for all packages :(

Reply:
The point here is to use this readme.md for nuget package description. We can add these labels to a separate .md file if needed.

Reply:
I usually check the nuget.org itself, since it's closer to the source of truth rather than looking through the labels.